A <%= exception.class.to_s %> occured: <%= exception.message.to_s %>
<% if exception.respond_to?(:backtrace) && exception.backtrace.present? %>
File:                    <%=   exception.backtrace.slice(0, 1).join("\n  ")  %>
<% end %>
<% if request_data.present? %>
IP:                      <%= request_data[:ip_address] %>
<% end %>
<% if request_data.present? %>
Session:                      <%= request_data[:session]%>
<% end %>
<% if request_data.present? %>
Request URI:             <%= request_data[:url] %>
<% end %>
<% if request_data.present? %>
Referer:                 <%= request_data[:referrer] %>
<% end %>
PID:                     <%= process %>
PWD:                     <%= pwd %>
Uname:                   <%= uname %>
Server:                  <%= server %>
Error time:	             <%= timestamp %>
<% if request_data.present? %>
User Agent:	             <%= request_data[:user_agent] %>
<% end %>
